systemtest/src/main/java/io/strimzi/systemtest/kafkaClients/api/ClientArgumentMap.java

```diff
@@ -2,7 +2,7 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-package io.strimzi.systemtest.clients.api;
+package io.strimzi.systemtest.kafkaClients.api;
```

Comment:
java package names should be names [all lower case](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html).